src/eval/index.php

```diff
@@ -1,9 +1,12 @@
 <?php
+	// set me to false to turn off sandboxing
+	$sandbox_me = true;
```

Comment:
Thanks for clarifying. For user ease, I was thinking:

```php
  $sandbox_me = isset($_ENV['DEP_NAME']);
```

That env var (name is at the mercy of cloudcontrol, unfortunately) wouldn't need to be set by users (unless they want to force local sandboxing). That var would be set for all production hosting environments though (cloudcontrol and heroku).  

This way, a user simply has to download the project, serve locally, and then visit the local hostname for an unsandboxed experience – no fiddling with the code is necessary. It also makes for a seamless upgrade path.

I'll merge this in as is and, if you agree, change it to the env var solution in a follow up commit.